### PR TITLE
[CI:DOCS] Update buildah-build.1.md

### PR DESCRIPTION
**Pull request by dandan2000**
The description of squash is not correct. I (tried) to explain it better.

Signed-off-by: dandan2000 <daniel.celentano@gmail.com>

#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
/kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### What this PR does / why we need it:

Just a clarification over squash behavior

#### How to verify it

-

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Review by flouthoc — CHANGES_REQUESTED** [review]
Changes looks correct to me just formatting issues, @dandan2000 could you please check failing smoke test :)

**Comment by dandan2000** [comment]
@flouthoc smoke test fixed.

**Review by flouthoc — APPROVED** [review]
@dandan2000 Thanks LGTM :) @containers/buildah-maintainers PTAL

File: docs/buildah-build.1.md
Comment (rhatdan) [inline docs/buildah-build.1.md:713]: I don't like the We, should be buildah.  Also this sentence does not make sense.

--squash defaults to "false"  So we don't squash image by default?

Comment (flouthoc) [reply docs/buildah-build.1.md:713]: Buildah by default does not create layers and puts everything in a single layer unless `--layer=true` but keeps the base layer separately but using `--squash=true` ensures that all base layers and and new layers are squashed into one.

Comment (rhatdan) [reply docs/buildah-build.1.md:713]: When I read this I come away with the understanding that

from fedora
run touch /foobar

Would create a squashed image of fedora and /foobar in an image as opposed to  the fedora image plus a layer with the file /foobar in it.

**Comment by rhatdan** [comment]
@TomSweeneyRedHat @edsantiago PTAL

File: docs/buildah-build.1.md
Comment (edsantiago) [inline docs/buildah-build.1.md:713]: Thank you for taking this on, @dandan2000. I think I can see what you're trying to say, but unfortunately it does not parse grammatically. And there's no `--layer` option (the name is `--layers`, with an `s`). Would this wording address your concerns instead?

>Squash all layers, including those from base image(s), into one single layer. (Default is `false`).
>
>By default, Buildah preserves existing base-image layers and adds only one new layer on a build.
>The `--layers` option can be used to preserve intermediate build layers.

**Comment by edsantiago** [comment]
Thanks. Could you please squash your commits before we ask for final review?

**Comment by rhatdan** [comment]
@dandan2000 
git rebase -i origin
SQUASH COMMITS
git push --force

**Comment by edsantiago** [comment]
@dandan2000 something screwed up badly here, git-wise. Looks like you maybe picked up parts of other commits.

**Comment by dandan2000** [comment]
> @dandan2000 something screwed up badly here, git-wise. Looks like you maybe picked up parts of other commits.

I think it is fixed now, I squashed all my commits and I synced the branch with main.

**Comment by edsantiago** [comment]
There's still a merge commit :(

**Comment by dandan2000** [comment]
> There's still a merge commit :(

Squashed, could you check it now please?

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by rhatdan** [comment]
/lgtm
Thanks @dandan2000 for the PR and your perseverance.

**Comment by edsantiago** [comment]
/hold

looks like CI is stuck, some sort of "workflow awaiting approval" thing. I'll approve and see what happens.

**Comment by edsantiago** [comment]
/hold cancel

thanks again @dandan2000

**Pull request merged**
